src/Label/Required.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import propTypes from '@dhis2/prop-types'
+import { spacers } from '../theme.js'
+
+export const Required = ({ dataTest }) => (
+    <span data-test={dataTest}>
+        *
+        <style jsx>{`
+            span {
+                padding-left: ${spacers.dp4};
+            }
+        `}</style>
+    </span>
+)
+
+Required.propTypes = {
+    dataTest: propTypes.string.isRequired,
```

Comment:
No default value for the `dataTest` prop in this case? Maybe not needed as this is an internal component?

Reply:
Yeah for the child data-test attributes (like the buttons in SplitButton, or the remove icon in the Chip), I've not set a default since they're always a child of another component, and that component should set the data-test attribute.

Reply:
But since it has to be set I've made it required for all of them.

Reply:
Cool, thanks for explaining the reason.